app/helpers/measures_helper.rb

```diff
@@ -33,4 +33,28 @@ def check_how_to_export_goods_link(declarable:, country_code:, country_name:, eu
       link_to('Check how to export commodity goods (link opens in new tab)', TradeTariffFrontend.check_duties_service_url, target: '_blank', rel: 'noopener')
     end
   end
+
+  def reformat_duty_expression(expression)
+    # meursing measures are wrapped with strong tags it seems
+    wrapping_tag = expression.start_with?('<strong>') ? :strong : :span
+    sanitized = sanitize(expression, tags: %w[abbr], attributes: %w[title])
+    components = []
+
+    while sanitized.length.positive?
```

Comment:
This feels unrubyish to me. Feels like we could handle splitting and iterating with each_wiith_object, etc

Reply:
Spoken about this offline. The split implementation doesn't yield what was split on so we'd struggle to recreate this behaviour